packages/vant/src/action-bar/README.md

```diff
@@ -90,7 +90,7 @@ Use `badge` prop to show badge in icon.
 | Attribute | Description | Type | Default |
 | --- | --- | --- | --- |
 | safe-area-inset-bottom | Whether to enable bottom safe area adaptation | _boolean_ | `true` |
-| placeholder | Whether to generate a placeholder element | _boolean_ | `false` |
+| placeholder | Whether to generate a placeholder element | _string_ | `false` |
```

Comment:
I think `boolean` is correct, see https://github.com/youzan/vant/blob/main/packages/vant/src/action-bar/ActionBar.tsx#L11

Reply:
I instinctively think that placeholder should be of type string. hhh